e2e/cypress/integration/api/planetsRouter.spec.js

```diff
@@ -0,0 +1,126 @@
+/// <reference types="Cypress" />
+
+context("/api/v1/planetsRouter" , () => {
+
+  describe("GET /planets", () => {
+  const initialPlanets = [{name:"Mercury", description:"very close to the sun"},{name:"Mars", description:"might have water"}]
+    
+  before(() =>{ 
+    cy.task("db:truncate", "Planet")
+    cy.task("db:insert", {modelName:"Planet", json:initialPlanets})
+  })
+
+  it("has the correct response type", () => {
+    cy.request("/api/v1/planets")
+    .its("headers")
+    .its("content-type")
+    .should("include", "application/json")
+  })
+
+  it("returns the correct status code", () =>{
+    cy.request("/api/v1/planets")
+    .its("status")
+    .should("be.equal", 200)
+  })
+
+  it("loads two planets", () =>{
+    cy.request("/api/v1/planets")
+    .its("body")
+    .its("planets")
+    .should("have.length", 2)
+  })
+
+  it("has the right properties for a planet", () =>{
+    cy.request("api/v1/planets")
+    .its("body")
+    .its("planets")
+    .should((planet)=>{
+      expect(planet[0]).to.have.property("name","Mercury")
+      expect(planet[0]).to.have.property("description", "very close to the sun")
+    })
+  })
+})
+
+  describe("GET /planets/:id", () =>{
+    const initialPlanet = {name:"Venus", description:"gas planet"}
+    let showPageUrl
+
+    before(() =>{ 
+      cy.task("db:truncate", "Planet")
+      cy.task("db:insert", {modelName:"Planet", json:initialPlanet})
+      cy.task("db:find", {modelName:"Planet",conditions:{name:"Venus"}}).then((planet) => {
+        showPageUrl = `/api/v1/planets/${planet[0].id}`
+      })
+    })
+
+    it("has the correct response type", () => {
+      cy.request(showPageUrl)
+      .its("headers")
+      .its("content-type")
+      .should("include", "application/json")
+    })
+  
+    it("returns the correct status code", () =>{
+      cy.request(showPageUrl)
+      .its("status")
+      .should("be.equal", 200)
+    })
+  
+    it("returns the correct data", () =>{
+      cy.request(showPageUrl)
+      .its("body")
+      .its("planet")
+      .should("have.property", "name","Venus")
```

Comment:
Im very okay with you incrementally adding tests. However, in this case, this test we aren't getting a representative data set. Usually for this endpoint, we also get all of the reviews and votes. 

If you want me to merge this in, it has to check for review information and upVotes and downVotes. this means updating your `beforeEach` quite a bit unfortunately. You'll need to add your planet, and then in a `.then` (similar to what you have on lines 51-52, add a review, and then finally `.then` to add at least one vote (though ideally ~3 would be nice)
finally, test for that data

If you would like, you can pause on this PR for a moment, work on something else, and come back to finish it